src/roman.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
Please remove this import and all references to `print_function`, just use the `print` function directly, which is fully supported in the lowest Python version this package supports (2.7)

Reply:
`print` is only a function in Python 2.7 with that import. Otherwise it's still a statement.

Reply:
```python
$ bin/python2.7
Python 2.7.17 (default, Jun 18 2020, 16:46:00)
[GCC 4.2.1 Compatible Apple LLVM 11.0.3 (clang-1103.0.32.62)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> print('foo')
foo
>>>
```

Reply:
That's a common misunderstanding (and a trick commonly used in doctests!). A one-element tuple *looks* like a function call and appears to behave like one, but as soon as you add other elements, you see the difference. (The lack of a return value is also a give-away.)

```pycon
Python 2.7.14 (default, Mar 14 2018, 13:36:31)
[GCC 7.3.1 20180303 (Red Hat 7.3.1-5)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> print('foo')
foo
>>> print('foo', 'bar')
('foo', 'bar')
>>> print

>>>
```

compared to
```pycon
Python 3.8.3 (default, May 16 2020, 08:39:55)
[Clang 11.0.3 (clang-1103.0.32.29)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> print('foo')
foo
None
>>> print('foo', 'bar')
foo bar
None
>>> print
<built-in function print>
>>>
```

Reply:
You're right. Learn something new every day 😄